reposilite-frontend/src/components/browser/Card.vue

```diff
@@ -225,4 +254,41 @@ export default {
 .token.string {
     color: mediumpurple;
 }
+.config-dropdown {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  margin: 24px 0;
+}
+.config-dropdown ul {
+  display: none;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 10px;
+  background-color: #e5e5e56e;
+}
+.selected-config {
+  width: 100%;
+  padding: 5px 10px;
+  border-radius: 8px;
+  border: 1px solid #e5e5e5;
+  box-sizing: border-box;
+}
+.dropdown-icon {
+  width: 20px;
+  height: 25px;
+  float: right;
+  margin: auto;
+  display: flex;
+  align-items: center;
+}
+@media screen and (max-width: 576px ) {
+  .config-selector-tab {
+    display: none !important;
+  }
+  .config-dropdown {
+    display: flex;
+  }
+}
```

Comment:
In general we prefer to use WindiCSS that targets responsive properties with prefixes as described below:

* https://windicss.org/features/responsive-design.html

For instance, `text-black lg:text-white` makes text by default black, but for higher screens it makes text white :)

Reply:
Done, Changed to windicss media query classes.